### DISCUSSION TITLE
unable to connect my bluetooth keyboard.

### DISCUSSION THREAD

**Discussion by arashutosh** [blueman-project/blueman#2034] [Q&A] [answered]
my bluetooth keyboard is not showing in bluetooth device list in mint operating  system. need some guidance in it.
other devices are able to conect normally like my smartphone connects smoothly but the keyboard is not able to show in the devices list.

**Comment by cschramm** [comment]
Which keyboard is it?

Try disabling `Hide unnamed devices` in the `View` menu.

Double check if it's visible from another device (e.g. your smartphone).

**Reply by arashutosh** [reply]
my keyboard is Targus KB55 AKB55TT.
yes, it is working on my smartphone and other computers. when I'm trying to connect it with windows it connects but when i was trying to connect it in linux mint then it doesn't even shows in available devices.
looking for a solution for this.
thanks for your response

**Reply by arashutosh** [reply, marked as answer]
My keyboard is able to connect now in my system. when i disabled the hide unnamed devices in the view menu, it worked.

thank you so much for your time.
